src/Samples/WinUIDesktopSample/WinUIDesktopSample.csproj

```diff
@@ -11,7 +11,8 @@
     -->
     <XamlCodeGenerationControlFlags>DoNotGenerateOtherProviders</XamlCodeGenerationControlFlags>
     <Platforms>x86;x64</Platforms>
-    <MsAppxPackageTargets>$(MSBuildThisFileDirectory)Net5_WinUI_Fixup.Targets</MsAppxPackageTargets>
+    <TargetPlatformIdentifier>Windows</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
```

Comment:
Do these do the same thing as if you had `net5.0-windows10.0.19041.0` in your `TargetFramework` node ?

Reply:
that's even better - thx!